lib/network_manager.go

```diff
@@ -370,6 +395,16 @@ func (nm *NetworkManager) processInboundConnection(conn Connection) (*RemoteNode
 		return nil, fmt.Errorf("NetworkManager.handleInboundConnection: Connection is not an inboundConnection")
 	}
 
+	na, err := nm.ConvertIPStringToNetAddress(ic.connection.RemoteAddr().String())
+	if err != nil {
+		return nil, errors.Wrapf(err, "NetworkManager.handleInboundConnection: Problem calling "+
+			"ConvertIPStringToNetAddress for addr: (%s)", ic.connection.RemoteAddr().String())
+	}
+	//if !addrmgr.IsRoutable(na) {
```

Comment:
This change around excluding IsRoutable addresses was reverted so you can delete it. With the update to use external addresses it could actually be a good check, but it's not worth putting it through testing to figure that out so just delete.

You can also move the call to ConvertIPStringToNetAddress back to where it was.